src/store.js

```diff
@@ -99,6 +100,7 @@ export const store = {
   },
   async initMainSettings(chainId, rskConfig, sideConfig) {
     const state = store.state
+    state.gasToken = 'BTC' // TODO: In the future, the UI will support other tokens to pay for the gas. Currently this is only possible using rBTC. 
```

Comment:
gasToken depends on the chainId, if you are in Ethereum is ETH. If it's on RSK is BTC if it's on Binance is BNB

Reply:
fixed.